daemon/gluster-blockd.c

```diff
@@ -333,9 +334,15 @@ main (int argc, char **argv)
     exit(EXIT_FAILURE);
   }
 
+  gbCfg = glusterBlockSetupConfig(NULL);
+  if (!gbCfg) {
+    LOG("mgmt", GB_LOG_ERROR, "%s", "glusterBlockSetupConfig() failed");
+    return -1;
+  }
+
   if (glusterBlockDParseArgs(argc, argv)) {
```

Comment:
Yeah, this still the same as before:
1, when the options are commented in /etc/sysconfig/gluster-blockd, it will use the value in gluster-blockd.service's:

 10 Environment="GB_GLFS_LRU_COUNT=5"
 11 Environment="GB_LOG_LEVEL=INFO"

And in /etc/sysconfig/gluster-blockd:
 # 10 GB_GLFS_LRU_COUNT=115                                                                                                                                                    
 # 16 GB_LOG_LEVEL=TRACE

● gluster-blockd.service - Gluster block storage utility
   Loaded: loaded (/usr/local/lib/systemd/system/gluster-blockd.service; enabled; vendor preset: disabled)
   Active: active (running) since Fri 2018-09-21 01:11:58 EDT; 4s ago
 Main PID: 11064 (gluster-blockd)
   CGroup: /system.slice/gluster-blockd.service
           └─11064 /usr/local/sbin/gluster-blockd --glfs-lru-count 5 --log-level INFO


2, if we uncomment the options in /etc/sysconfig/gluster-blockd, it will use the values in in gluster-blockd.service instead:

 10 GB_GLFS_LRU_COUNT=115                                                                                                                                                    
 16 GB_LOG_LEVEL=TRACE

● gluster-blockd.service - Gluster block storage utility
   Loaded: loaded (/usr/local/lib/systemd/system/gluster-blockd.service; enabled; vendor preset: disabled)
   Active: active (running) since Fri 2018-09-21 01:15:28 EDT; 2s ago
 Main PID: 11162 (gluster-blockd)
   CGroup: /system.slice/gluster-blockd.service
           └─11162 /usr/local/sbin/gluster-blockd --glfs-lru-count 115 --log-level TRACE

Reply:
Okay. That should be good.